CMakeLists.txt

```diff
@@ -1222,13 +1222,14 @@ set(BUILD_PT_POST_PROCESSOR OFF)
 set(proc_supports_pt OFF)
 # Right now we only build PT related libraries on Linux x86_64.
 if (LINUX AND X86 AND X64)
-  # We will always build the PT tracer no matter whether the system supports PT.
+  # We will always build the PT tracer on Linux X86_64 no matter whether the system supports PT.
+  # Because users may want to use the built binaries on a different system that does support PT.
```

Comment:
Do not make this a separate sentence: s/PT. Because/PT, because/

Reply:
Done.